tests/release/pipelines/push_to_external_registry.go

```diff
@@ -30,7 +30,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-to-pyxis", Label("release-pipelines", "pushPyxis", "HACBS"), Pending, func() {
+var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-to-pyxis", Label("release-pipelines-pyxis", "pushPyxis", "HACBS"), func() {
```

Comment:
shouldn't this still be the `release-pipelines` label? Or else it won't run with the `release-pipelines` suite IIUC

Reply:
Just for testing before merge ...